tamr_toolbox/models/project_artifacts.py

```diff
@@ -0,0 +1,162 @@
+""" Project artifacts data classes """
+from dataclasses import dataclass
+
+
+@dataclass()
+class SchemaMappingArtifacts:
```

Comment:
Notice that you need to specify the type as str for each field. For example:
`UNIFIED_ATTRIBUTES: str = "UNIFIED_ATTRIBUTES"`

If you don't then you can't loop over the fields. See this for example
```
In [11]: from dataclasses import dataclass, fields                                                                                                                                                            

In [12]: @dataclass 
    ...: class A: 
    ...:   x: int = 1 
    ...:   y: int = 2 
    ...:                                                                                                                                                                                                      

In [13]: [field.name for field in fields(A)]                                                                                                                                                                  
Out[13]: ['x', 'y']

In [14]: @dataclass 
    ...: class A: 
    ...:   x = 1 
    ...:   y = 2                                                                                                                                                                                              

In [15]: [field.name for field in fields(A)]                                                                                                                                                                  
Out[15]: []
```

Reply:
Fixed